staging/src/k8s.io/legacy-cloud-providers/aws/aws_fakes.go

```diff
@@ -305,6 +305,12 @@ func (ec2i *FakeEC2Impl) ModifyInstanceAttribute(request *ec2.ModifyInstanceAttr
 	panic("Not implemented")
 }
 
+// ModifyNetworkInterfaceAttribute is not implemented but is required for interface
+// conformance
+func (ec2i *FakeEC2Impl) ModifyNetworkInterfaceAttribute(request *ec2.ModifyNetworkInterfaceAttributeInput) (*ec2.ModifyNetworkInterfaceAttributeOutput, error) {
+	panic("Not implemented")
```

Comment:
Can we add a test case that actually exercises this call?

Reply:
added.